far/vmenu.hpp

```diff
@@ -118,8 +118,24 @@ struct menu_item
 	LISTITEMFLAGS SetGrayed(bool Value) { if (Value) Flags|=LIF_GRAYED; else Flags&=~LIF_GRAYED; return Flags;}
 };
 
-struct MenuItemEx: menu_item
+struct menu_item_ex: menu_item
 {
+	menu_item_ex() = default;
+	MOVABLE(menu_item_ex);
+
+	explicit menu_item_ex(const menu_item& Item)
+		: menu_item{ Item }
+	{}
+
+	explicit menu_item_ex(LISTITEMFLAGS Flags)
+		: menu_item{ string{}, Flags }
+	{}
+
+	template<typename T> requires (std::is_constructible_v<menu_item, T, LISTITEMFLAGS>)
```

Comment:
I removed all these ctors in 168c644 because I wanted to avoid copying, but correct handling of strings and string_views was getting annoyingly complex.
Looks like you managed to overcome it with this is_constructible_v trick?

Reply:
I hope so. 🙂

Reply:
Made it non-copyable, because why not.